Random Users/Model/User.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  User.swift
+//  Random Users
+//
+//  Created by Chad Parker on 5/15/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import Foundation
+
+struct User: Codable {
+    
+    let name: Name
+    struct Name: Codable {
+        let title: String
+        let first: String
+        let last: String
+    }
+    
+    let email: String
+    let phone: String
+    
+    let picture: Picture
+    struct Picture: Codable {
+        let large: URL
+        let medium: URL
+        let thumbnail: URL
+    }
+    
```

Comment:
I would just order this around like group the structs together and then group the lets together.